commands/genman.go

```diff
@@ -36,7 +35,7 @@ in the "man" directory under the current directory.`,
 		header := &doc.GenManHeader{
 			Section: "1",
 			Manual:  "Hugo Manual",
-			Source:  fmt.Sprintf("Hugo %s", helpers.HugoVersion()),
+			Source:  "Hugo " + helpers.HugoVersion(),
```

Comment:
Yes😅
But I think, it's nicer and less to read (and a little bit faster too). But if you don't like it, I can revert it.